src/main/java/at/xirado/bean/command/handler/SlashCommandHandler.java

```diff
@@ -138,18 +138,14 @@ private void registerCommand(SlashCommand command)
     {
         if (!command.isGlobal() && !Bean.getInstance().isDebug())
         {
-            if (command.getEnabledGuilds() == null) return;
-            if (command.getEnabledGuilds().isEmpty()) return;
+            if (command.getEnabledGuilds() == null || command.getEnabledGuilds().isEmpty()) return;
             for (Long guildID : command.getEnabledGuilds())
             {
                 Guild guild = Bean.getInstance().getShardManager().getGuildById(guildID);
                 if (guild == null) continue;
                 List<SlashCommand> alreadyRegistered = registeredGuildCommands.containsKey(guildID) ? registeredGuildCommands.get(guildID) : new ArrayList<>();
                 alreadyRegistered.add(command);
                 if (registeredGuildCommands.containsKey(guildID))
-                {
-                    registeredGuildCommands.replace(guildID, alreadyRegistered);
-                } else
                 {
                     registeredGuildCommands.put(guildID, alreadyRegistered);
                 }
```

Comment:
This check is wrong

Reply:
I am not sure what you mean by this. It does always put them in the map. I just removed HashMap#replace because put will replace if it exist

Reply:
No it won't always put them in the map, only if `containsKey` returns true. Just extract `put` from the if statement

Reply:
Changed!